File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/AssetReeferResponseReeferStatsSetPoint.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param tempInMilliC Set point temperature in millidegree Celsius.
 * @param changedAtMs Timestamp in Unix milliseconds since epoch.
 */
data class AssetReeferResponseReeferStatsSetPoint (

        @JsonProperty("tempInMilliC") val tempInMilliC: Long? = null,

        @JsonProperty("changedAtMs") val changedAtMs: Long? = null
) {

}

File: openapi-generator/dart-jaguar/lib/model/asset_reefer_response_reefer_stats_set_point.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

part 'asset_reefer_response_reefer_stats_set_point.jser.dart';

class AssetReeferResponseReeferStatsSetPoint {
   /* Set point temperature in millidegree Celsius. */
  @Alias('tempInMilliC')
  final int tempInMilliC;
   /* Timestamp in Unix milliseconds since epoch. */
  @Alias('changedAtMs')
  final int changedAtMs;
  

  AssetReeferResponseReeferStatsSetPoint(
      

{
     this.tempInMilliC = null,  
     this.changedAtMs = null 
    
    }
  );

  @override
  String toString() {
    return 'AssetReeferResponseReeferStatsSetPoint[tempInMilliC=$tempInMilliC, changedAtMs=$changedAtMs, ]';
  }
}

@GenSerializer(nullableFields: true)
class AssetReeferResponseReeferStatsSetPointSerializer extends Serializer<AssetReeferResponseReeferStatsSetPoint> with _$AssetReeferResponseReeferStatsSetPointSerializer {

}
